### PR TITLE
Mark optional property with square brackets

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

**Comment by tschaub** [comment]
Will this also require work on the tools that create the typedefs for the Compiler?

**Comment by fredj** [comment]
I think it's only for the api doc because the type are already optional (with the `undefined` type) but that's only a supposition.

**Comment by ahocevar** [comment]
> Will this also require work on the tools that create the typedefs for the Compiler?

The babel-plugin-jsdoc-closure supports this already.

File: src/ol/View.js [hunk 1]
```diff
@@ -146,9 +146,8 @@ import Units from './proj/Units.js';
  * the animation.
  * @property {ol.Coordinate|undefined} anchor Optional anchor to remained fixed
  * during a rotation or resolution animation.
- * @property {number|undefined} duration The duration of the animation in milliseconds
- * (defaults to `1000`).
- * @property {undefined|function(number):number} easing The easing function used
+ * @property {number|undefined} [duration=1000] The duration of the animation in milliseconds.
+ * @property {undefined|function(number):number} [easing] The easing function used
```
Comment (ahocevar) [inline src/ol/View.js]: Oh sorry for the inaccurate review. You should have removed the `undefined` in all places where you set the type in square parentheses. The babel-plugin-closure-jsdoc will resolve that to the correct type.

**Comment by ahocevar** [comment]
@fredj Can you please create a follow-up pull request to address https://github.com/openlayers/openlayers/pull/7975#discussion_r174076795? Thanks.

File: src/ol/View.js (hunk 1 above)
Comment (fredj) [reply src/ol/View.js]: see #7979

**Pull request merged**